src/content/docs/synthetics/synthetic-monitoring/administration/user-roles-synthetic-monitoring.mdx

```diff
@@ -41,6 +42,4 @@ To get started, follow [standard procedures to create a custom role](/docs/accou
 
 ## Users on New Relic One user model [#new-pricing]
 
-For users on the New Relic One user model, the ability to use the synthetic monitoring UI to create and manage synthetic monitors requires the [full platform user](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) user type.
-
-For limitations related to API usage, see [User limitations](/docs/accounts/accounts-billing/new-relic-one-pricing-users/transition-guide-new-pricing-plan#limitations).
+Use of the synthetic monitoring UI requires a user type of [full platform user](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) user type. For information on roles and capabilities, see [Capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#capabilities).
```

Comment:
A little weird. I'd use "Use of the synthetic monitoring UI requires a [full platform user](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) user type."